BREAKING.md

```diff
@@ -189,6 +190,10 @@ Additionally, please note that the `Connecting` state is being renamed to `Catch
 `ConnectionState.Connecting` is marked as deprecated, please use `ConnectionState.CatchingUp` instead.
 `ConnectionState.Connecting` will be removed in the following major release.
 
+### Remove ICodeLoader from container-definitions
+`ICodeLoader` in `@fluidframework/container-definitions` was deprecated a while ago and is now removed. Use `ICodeDetailsLoader` by importing it from `@fluidframework/container-loader`.
```

Comment:
Nits:

> `ICodeLoader` in `@fluidframework/container-definitions` was deprecated a while ago and is now removed.

Can we replace "a while ago" with the exact version? Either that or just remove it all together

```suggestion
`ICodeLoader` in `@fluidframework/container-definitions` was deprecated a while ago and is now removed. Use `ICodeDetailsLoader` from `@fluidframework/container-loader` instead.
```